tiny_gltf.h

```diff
@@ -2629,11 +2645,19 @@ bool LoadImageData(Image *image, const int image_idx, std::string *err,
   // the Image metadata to signal that this image uses 2 bytes (16bits) per
   // channel:
   if (stbi_is_16_bit_from_memory(bytes, size)) {
-    data = reinterpret_cast<unsigned char *>(
-        stbi_load_16_from_memory(bytes, size, &w, &h, &comp, req_comp));
-    if (data) {
-      bits = 16;
-      pixel_type = TINYGLTF_COMPONENT_TYPE_UNSIGNED_SHORT;
+    if (option.as_is) {
```

Comment:
This needs some rework before merging.

Reply:
Thanks!

Reply:
Done. Please review.

Reply:
Thanks. Let me give some time to review it

Reply:
In default `LoadImageData`, even as_is flag is set, it tries to decode image header(width, height, ...)
Can we make loading image header optional when `as_is` flag is set?
i.e., when failed to parse image header using STBI,  fill image buffer but set invalid size(default value) to other parameters(width, height, ...), report warning, and return true;

Reply:
I can do that. Tbhat leaves the question what to do if we load the image from a buffer, i.e. req_width != 0 and req_height != 0? Just warn for this case., too? Or error out if we parse the image header held by a glTF buffer?

Reply:
Done